commons/src/main/java/org/eclipse/kapua/commons/jpa/ResourceSqlScriptExecutor.java

```diff
@@ -89,8 +89,8 @@ public int executeUpdate(EntityManager entityManager) {
             StringBuilder sqlStringBuilder = new StringBuilder();
             try {
                 try (
-                        InputStream inputStream = getClass().getClassLoader().getResourceAsStream(qStr);
-                        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream))) {
+                    InputStream inputStream = Thread.currentThread().getContextClassLoader().getResourceAsStream(qStr);
```

Comment:
Please update the header to "2017, 2020".

Reply:
Hi, do you perhaps mean this at this [line](https://github.com/eclipse/kapua/blob/5246cd29e5ea87aee71374f262d6d07c38071b2d/commons/src/main/java/org/eclipse/kapua/commons/jpa/ResourceSqlScriptExecutor.java#L2) ?

Reply:
yes, exactly! 😄